test-openvpn.ps1

```diff
@@ -35,12 +35,16 @@ Function Verify-Path ([string]$mypath) {
     }
 }
 
+$exe_path = (Get-ItemProperty 'HKLM:\SOFTWARE\OpenVPN' -Name 'exe_path').exe_path
+
 if (! $Openvpn) {
```

Comment:
I agree that that way of handling default values for parameters is cleaner. Most of the time we can safely assume the default values can be obtained (e.g. exe_path exists in registry). I would refactor parameter handling in another PR, though.

I think we should do a minimal environment check quite early on. For example, check if the OpenVPN registry key exists, and if not, fail immediately.